play-json-jsoniter/src/test/scala/com/evolutiongaming/jsonitertool/TestDataGenerators.scala

```diff
@@ -0,0 +1,138 @@
+package com.evolutiongaming.jsonitertool
+
+import java.util.concurrent.ThreadLocalRandom
+
+import play.api.libs.json.Json
+import valuegen.Preamble._
+import valuegen.{JsArrayGen, JsObjGen}
+import value.JsObj
+import org.scalacheck.{Arbitrary, Gen}
+
+object TestDataGenerators extends PlayJsonImplicits {
+
+  val ALPHABET: Vector[String] = "abcdefghijklmnopqrstuvwzyz".split("").toVector
+
+  def str(len: Int): Gen[String] = Gen.listOfN(len, Gen.choose(0, ALPHABET.size - 1).map(ALPHABET(_))).map(_.mkString(""))
+
+  def flagGen: Gen[Boolean] = Arbitrary.arbitrary[Boolean]
+
+  def nameGen: Gen[String] = str(ThreadLocalRandom.current.nextInt(5, 20))
```

Comment:
Would you consider `Gen.choose(5, 20).flatMap(str)` ?

Reply:
Ok